Sources/StreamVideo/WebRTC/WebRTCClient.swift

```diff
@@ -377,6 +424,44 @@ class WebRTCClient: NSObject {
             }
         }
     }
+        
+    private func handleOnMigrationJoinResponse() {
+        signalChannel?.connectionStateDelegate = nil
+        signalChannel?.onWSConnectionEstablished = nil
+        signalChannel?.disconnect {}
+        signalChannel = nil
+        signalChannel = migratingWSClient
+        if let migratingSignalService {
+            signalService = migratingSignalService
+            sfuMiddleware.signalService = migratingSignalService
+        }
+        cleanupMigrationData()
+        Task {
+            temp = subscriber
```

Comment:
Why are temporarily storing the subscriber to temp but below we are updating onConnected on subscriber? Shall we also cleanUp temp after we are done?

Reply:
hmm, not sure I follow this. In the `setupPeerConnections` we are creating a new `subscriber`.

Reply:
> In the setupPeerConnections we are creating a new subscriber.

That answers my question. Thanks :)